python/Ganga/GPIDev/Schema/Schema.py

```diff
@@ -402,7 +402,12 @@ def __len__(self):
     # item.isA(SimpleItem)
     def isA(self, _what):
 
-        from Ganga.GPIDev.Base.Proxy import stripProxy
+        try:
+            from Ganga.GPIDev.Base.Proxy import stripProxy
+        except ImportError:
+            def dummyfunc(arg):
+                return arg
+            stripProxy = dummyfunc
```

Comment:
If you take this out launch ganga. This fails due to a circular dependency. Proxy->GangaList->Schema->Proxy I think which emphasizes that we should just disallow proxies objects getting into the Schema by design imo.
I'd be happy to remove the stripProxy as long as we have a test which throws an exception if someone tries to stick a proxied object into the schema.